lib/ansible/modules/cloud/google/drive_append_to_sheet.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Chris Blum <chris.n.blum@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1', 'status': ['preview'], 'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: drive_append_to_sheet
+
+short_description: Archive variables in a Google Sheet
+
+version_added: "2.8"
+
+description:
+    - "Using this module it is possible to archive variables in a playbook run into a Google Sheet.
+      This is especially interesting if you are trying to gather facts across lots of servers and
+      you will want to do some statistical analysis on that data later on.
+      A Google account and the Python client library for Google APIs are requirements.
+      Also you must have the Google Sheets API activated and valid credentials for this handy.
+      The necessary API steps can be done here:
+      https://developers.google.com/sheets/api/quickstart/python#step_1_turn_on_the"
+
+options:
+    columns:
+        description:
+            - This is the variable that should be appended to the GSheet
+        required: true
+        aliases: [ 'name' ]
+    sheetID:
+        description:
+            - This is the Sheet ID
+        required: true
+    client_id:
+        description:
+            - This is the Google OAuth client_id
+              It ends with .apps.googleusercontent.com
+        required: true
+    client_secret:
+        description:
+            - This is the Google OAuth client_secret
+        required: true
+    sheetName:
+        description:
+            - This is the Sheet inside of the GSheet that we will add information
+        required: false
+    range:
+        description:
+            - This is the range
+        default: 'A1:A1'
+        required: false
+
+requirements:
+    - "google-api-python-client"
+    - "google-auth-httplib2"
+    - "google-auth-oauthlib"
+
+author:
+    - Chris Blum (@mulbc)
+'''
+
+EXAMPLES = '''
+# Append one row with three columns
```

Comment:
This should be your name-value.
```suggestion
- name: Append one row with three columns
```